src/store/common/actions.ts

```diff
@@ -0,0 +1,5 @@
+import { createAction } from '@reduxjs/toolkit';
+import { TOGGLE_ANNOTATION_VISIBILITY, TOGGLE_ANNOTATION_MODE, Mode } from './types';
+
+export const toggleAnnotationVisibilityAction = createAction(TOGGLE_ANNOTATION_VISIBILITY);
```

Comment:
Does this need a generic type defined, like `createAction<Visibility>` or something?